DeleteUserDataActivity/backupAndDelete.ts

```diff
@@ -95,6 +104,36 @@ const executeRecursiveBackupAndDelete = <T>(
     )
   );
 
+const backupAndDeleteAuthenticationLockData = (
+  authenticationLockService: AuthenticationLockService,
+  userDataBackup: IBlobServiceInfo,
+  fiscalCode: FiscalCode,
+  data: ReadonlyArray<AuthenticationLockData>
+): TE.TaskEither<DataFailure, true> =>
+  pipe(
+    saveDataToBlob(userDataBackup, "access/authentication-locks.json", data),
```

Comment:
What if the file already exists? Would it be overwritten? Maybe we should use an unique name to keep safe the file generation on various delete requests and in Activity retries.